lib/core/Concurrency.cc

```diff
@@ -55,16 +56,16 @@ class CThreadPoolExecutor final : public CExecutor {
 class CExecutorHolder {
 public:
     CExecutorHolder()
-        : m_ThreadPoolSize{0}, m_Executor(std::make_unique<CImmediateExecutor>()) {}
+        : m_ThreadPoolSize(0), m_Executor(std::make_unique<CImmediateExecutor>()) {}
```

Comment:
Let's keep the type checking in place here:
```suggestion
        : m_ThreadPoolSize{0}, m_Executor(std::make_unique<CImmediateExecutor>()) {}
```